src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -213,9 +229,20 @@ $services.localization.render("contrib.xpoll.finish.message")
 
 .xpoll th {
   background-color: $theme.backgroundSecondaryColor;
+}
+
+.polls:last-of-type {
+  margin-top: 10px;
+}
+
+.noitems {
```

Comment:
Please don't write generic selectors for XWiki Skins classes. This will affect all the instances. 

If you want to affect only your macro, make a more specific selector, like ".polls-recent .noitems". 

Still I don't think .noitems class should be used for this use case.